artichoke-backend/src/extn/core/time/mruby.rs

```diff
@@ -112,10 +112,11 @@ unsafe extern "C" fn time_self_now(mrb: *mut sys::mrb_state, _slf: sys::mrb_valu
 }
 
 unsafe extern "C" fn time_self_at(mrb: *mut sys::mrb_state, _slf: sys::mrb_value) -> sys::mrb_value {
-    let args = mrb_get_args!(mrb, *args);
```

Comment:
eventually when we add support for all variants of `Time::at`, it will probably be easier to process the arguments as an iterator, but for now, this change is OK to me.